placesautocomplete/src/main/java/com/seatgeek/placesautocomplete/PlacesAutocompleteTextView.java

```diff
@@ -439,4 +443,23 @@ public void setLanguageCode(@Nullable String languageCode) {
         this.languageCode = languageCode;
         api.setLanguageCode(this.languageCode);
     }
+
```

Comment:
I mentioned it in the OP, but the background broke when I updated it. We don't seem to set the background anywhere, so the fix seemed non-trivial. I could spend the time debugging if we want though.